.landscaper/blueprint/deploy-executions.yaml

```
@@ -0,0 +1,36 @@
+deployItems:
+- name: deploy
+  type: landscaper.gardener.cloud/kubernetes-manifest
+  target:
+    name: {{ .imports.cluster.metadata.name }}
+    namespace: {{ .imports.cluster.metadata.namespace }}
+  config:
+    apiVersion: manifest.deployer.landscaper.gardener.cloud/v1alpha1
```

you should use `v1alpha2`
```suggestion
    apiVersion: manifest.deployer.landscaper.gardener.cloud/v1alpha2
```